dist/GaugeChart/index.js

```diff
@@ -271,24 +272,24 @@ var drawNeedle = function drawNeedle(resize, prevProps, props, width, needle, co
       var currentPercent = (0, _d.interpolateNumber)(prevPercent, percent);
       return function (percentOfPercent) {
         var progress = currentPercent(percentOfPercent);
-        return container.current.select(".needle path").attr("d", calculateRotation(progress, outerRadius, width));
+        return container.current.select(".needle path").attr("d", calculateRotation(progress, outerRadius, width, needleScale));
       };
     });
   } else {
-    container.current.select(".needle path").attr("d", calculateRotation(percent, outerRadius, width));
+    container.current.select(".needle path").attr("d", calculateRotation(percent, outerRadius, width, needleScale));
   }
 };
 
-var calculateRotation = function calculateRotation(percent, outerRadius, width) {
-  var needleLength = outerRadius.current * 0.55,
+var calculateRotation = function calculateRotation(percent, outerRadius, width, needleScale) {
+  var needleLength = outerRadius.current * needleScale,
       //TODO: Maybe it should be specified as a percentage of the arc radius?
   needleRadius = 15 * (width.current / 500),
       theta = percentToRad(percent),
       centerPoint = [0, -needleRadius / 2],
       topPoint = [centerPoint[0] - needleLength * Math.cos(theta), centerPoint[1] - needleLength * Math.sin(theta)],
       leftPoint = [centerPoint[0] - needleRadius * Math.cos(theta - Math.PI / 2), centerPoint[1] - needleRadius * Math.sin(theta - Math.PI / 2)],
       rightPoint = [centerPoint[0] - needleRadius * Math.cos(theta + Math.PI / 2), centerPoint[1] - needleRadius * Math.sin(theta + Math.PI / 2)];
-  var pathStr = "M ".concat(leftPoint[0], " ").concat(leftPoint[1], " L ").concat(topPoint[0], " ").concat(topPoint[1], " L ").concat(rightPoint[0], " ").concat(rightPoint[1]);
+  var pathStr = "M ".concat(leftPoint[0], " ").concat(leftPoint[1], " L ").concat(topPoint[0] * scale, " ").concat(topPoint[1] * scale, " L ").concat(rightPoint[0], " ").concat(rightPoint[1]);
```

Comment:
`scale` variable is undefined. Please fix it.

Reply:
Hi there. Created a new PR for that issue :)

https://github.com/Martin36/react-gauge-chart/pull/143

Reply:
It is merged now